content/en/docs/collector/deployment/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Deployment
+description: Patterns you can apply to run the OpenTelemetry collector
+weight: 2
+---
+
+The OpenTelemetry collector consists of a single binary which you can use in
+different ways, for different use cases. This section describes deployment
+patterns, their use cases along with pros and cons and best practices for
+collector configurations for cross-environment and multi-backend deployments.
+
+## Other information
```

Comment:
One question that has come up fairly frequently regarding collector deployments is "when/why should I use the [Collector Operator](https://github.com/open-telemetry/opentelemetry-operator)?"

Would be helpful for end-users if the intro mentioned how the operator (optionally) fits into k8s-based deployments.

Reply:
@mhausenblas, wdyt? I am always a fan of having inter-linked pages to point people to other resources (not the repo, but the docs for k8s operator at /docs/k8s-operator/), but I don't see yet how to incorporate that to this page.